portfolio/src/main/webapp/script.js

```diff
@@ -44,10 +44,14 @@ function getServerMessage() {
     }); 
 }
 
-
 /** Creates an <li> element containing text. */
 function createListComments(text){
     const liElement = document.createElement('li');
-    liElement.innerText = text;
+    liElement.className = 'text';
```

Comment:
You don't need this. Usually people do this when there's a css style bound to the class, but I don't see any css for this.

Reply:
Oh okay, that makes sense, thank you!